Proton/Sources/Core/PRTextStorage.m

```diff
@@ -18,35 +18,29 @@
 //  limitations under the License.
 //
 
-#import <Foundation/Foundation.h>
-#import <UIKit/UIKit.h>
 #import "PRTextStorage.h"
-#import "EditorContentName.h"
-
+#import "PREditorContentName.h"
 #import <Proton/Proton-Swift.h>
 
 @interface PRTextStorage ()
 @property (nonatomic) NSTextStorage *storage;
-
 @end
 
 @implementation PRTextStorage
 
-@synthesize storage = _storage;
-
 - (instancetype)init {
     if (self = [super init]) {
-        self.storage = [[NSTextStorage alloc] init];
+        _storage = [[NSTextStorage alloc] init];
```

Comment:
Properties setters could have logic so calling them in constructor is probably not what you want.